src/components/application_manager/rpc_plugins/vehicle_info_plugin/include/vehicle_info_plugin/vehicle_info_plugin.h

```diff
@@ -71,6 +73,9 @@ class VehicleInfoPlugin : public plugins::RPCPlugin {
 
  private:
   void DeleteSubscriptions(app_mngr::ApplicationSharedPtr app);
+  void SelectVehicleDataForUbsubscribe(
+      const app_mngr::ApplicationSharedPtr& app,
+      VehicleInfoSubscriptions& vd_for_unscribe);
```

Comment:
@IGapchuk name of parameters

Reply:
Done. vd_for_unscribe was renamed to vehical_data_for_unscribe.